frontend/src/actions/commonActions.js

```diff
@@ -106,3 +106,15 @@ export const calculateSummary = (countObj) => {
   };
   return summary;
 };
+
+export const filterOtherStatus = (filterData) => {
+  const keyWordArr = ["success", "failure"];
+
+  const statusValues =
+    filterData.find((item) => item.key === "jobStatus").value || [];
+
+  const filteredStatus = statusValues.filter(
```

Comment:
So `failed` is "other" rather than counting as a failure??

Reply:
No, even "failed" is treated as "failure". This can be referred in the `JOB_STATUS_MAP` in constants.py for the exact mapping. In the response, the `jobStatus` can be  success , failure or other values. So, in the actions filtering out success and failure